src/vs/editor/contrib/markdownRenderer/browser/markdownRenderer.ts

```diff
@@ -93,6 +94,10 @@ export class MarkdownRenderer {
 					element.style.fontFamily = this._options.codeBlockFontFamily;
 				}
 
+				if (this._options.codeBlockFontSize !== undefined) {
+					element.style.fontSize = this._options.codeBlockFontSize;
```

Comment:
I'd prefer not having the markdown render add styling like this  

Can the comment widget use css to override the default fontSize instead?

Reply:
@mjbvz I've switched over to a css approach instead now.

Reply:
Sorry if I'm being stupid here, but I don't understand why the markdown renderer needs to be modifiered. Can the review widget add the required css / classes instead of having the markdown renderer do it?

Reply:
The problem I see is this line:

https://github.com/microsoft/vscode/blob/92bf104b5294d976c0fcfbdc69e43ba70208b6e8/src/vs/editor/contrib/markdownRenderer/browser/markdownRenderer.ts#L91-L92

It add styles directly to the element, including font size. I'm not a css expert, but my understanding is that styles added to an element get precedence and you cannot override them.

Reply:
Ok thanks. I would not expect `codeBlockRenderer` to be invoked for inline code like those that #146183 seems to show. Is the issue with inline code or with actual code blocks?

Reply:
It's an issue with code blocks. I was able to use styles to deal with inline code, but not code blocks. I polled the team and the feedback was that code blocks and inline code should both be the same size as the surrounding non-code text in comments.

Reply:
Per discussion, I've switched back to the original approach.